docs/lib/analytics/fragments/ios/record.md

```diff
@@ -4,22 +4,21 @@ The Amplify Analytics plugin provides a simple interface to record custom events
 
 ```swift
 func recordEvents() {
-    let properties = ["eventPropertyStringKey": "eventProperyStringValue",
-                      "eventPropertyIntKey": 123,
-                      "eventPropertyDoubleKey": 12.34,
-                      "eventPropertyBoolKey": true] as [String: AnalyticsPropertyValue]
+    let properties: [String: AnalyticsPropertyValue] = [
+        "eventPropertyStringKey": "eventProperyStringValue",
+        "eventPropertyIntKey": 123,
+        "eventPropertyDoubleKey": 12.34,
+        "eventPropertyBoolKey": true
+    ]
     let event = BasicAnalyticsEvent("eventName", properties: properties)
     Amplify.Analytics.record(event: event)
-
-    // Plugin will automatically flush events. 
-    // You do not have to do this in the app code.
-    Amplify.Analytics.flushEvents() 
```

Comment:
:+1: This definitely doesn't need to be here, as the self-conscious comment suggests.